doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1581,71 +1548,65 @@ Numeral notations
      Both functions passed to :cmd:`Numeral Notation` must be single
      identifiers.
 
+     .. todo PR: in these 2 exns, how does "must be single identifiers" fit with their
+        declaration as qualid??
```

Comment:
The descriptions are misleading. Add "qualified" in front of "identifiers".

Reply:
I'm not sure the description is that misleading.  You get this error message if you try to pass an alias, notation, or term rather than a (qualified) identifier.  I'm not sure if you get this message if you pass an identifier that does not exist in the global environment.  Is there anything that's accepted by `ident` which is not accepted by `qualid`?

Reply:
So "single qualified identifiers"?